apps/omg_watcher/test/omg_watcher/exit_processor/exit_info_test.exs

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Watcher.ExitProcessor.ExitInfoTest do
+  @moduledoc """
+  Test of the logic of the exit_info module
+  """
+  use OMG.Watcher.ExitProcessor.Case, async: true
+
+  alias OMG.Watcher.ExitProcessor.ExitInfo
+
+  import OMG.Watcher.ExitProcessor.TestHelper
+
+  @recently_added_keys [:root_chain_txhash]
+  @utxo_pos_1 {0, 0, 0}
```

Comment:
Nah, this is special value. I'd like to have real-world values in tests
```suggestion
  @utxo_pos_1 {1000, 0, 0}
```

Reply:
Makes sense not to have a null value! `OMG.Utxo.positon` will also have the `:utxo_position` atom, so I just switched the `blknum` to 1000.